package.json

```diff
@@ -105,8 +105,8 @@
         "inquirer": "6.3.1"
     },
     "engines": {
-        "node": ">=12.16.1",
-        "npm": ">=6.13.4"
+        "node": ">=10.20.1",
```

Comment:
this change is the good one. This PR should only change these 2 values to be less restricted

Reply:
@pascalgrimaud : I've reverted the generator node version back to 12.16.1. Could you please let me know if everything looks okay now? I haven't changed the template or CI node versions [as described in the above comment](https://github.com/jhipster/generator-jhipster/pull/11687#discussion_r420035667) there seem to be differing opinions on that. If that is also needed let me know.

Reply:
good to go, thanks :)